docs/samples/Microsoft.ML.AutoML.Samples/BinaryClassificationExperiment.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using Microsoft.ML.Auto;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.AutoML.Samples
+{
+    public static class BinaryClassificationExperiment
+    {
+        private static string BaseDatasetsLocation = "Data";
+        private static string TrainDataPath = Path.Combine(BaseDatasetsLocation, "wikipedia-detox-250-line-data.tsv");
+        private static string TestDataPath = Path.Combine(BaseDatasetsLocation, "wikipedia-detox-250-line-test.tsv");
+        private static string ModelPath = Path.Combine(BaseDatasetsLocation, "SentimentModel.zip");
+        private static uint ExperimentTime = 60;
+
+        public static void Run()
+        {
+            MLContext mlContext = new MLContext();
+
+            // STEP 1: Load data
+            IDataView trainDataView = mlContext.Data.LoadFromTextFile<SentimentIssue>(TrainDataPath, hasHeader: true);
+            IDataView testDataView = mlContext.Data.LoadFromTextFile<SentimentIssue>(TestDataPath, hasHeader: true);
+
+            // STEP 2: Run AutoML experiment
+            Console.WriteLine($"Running AutoML binary classification experiment for {ExperimentTime} seconds...");
+            ExperimentResult<BinaryClassificationMetrics> experimentResult = mlContext.Auto()
+                .CreateBinaryClassificationExperiment(ExperimentTime)
+                .Execute(trainDataView);
+
+            // STEP 3: Print metric from the best model
+            RunDetail<BinaryClassificationMetrics> bestRun = experimentResult.BestRun;
+            Console.WriteLine($"Total models produced: {experimentResult.RunDetails.Count()}");
+            Console.WriteLine($"Best model's trainer: {bestRun.TrainerName}");
```

Comment:
I'm not sure how to best do it, but printing the featurization would be nice.

Reply:
Great point. +1 that not sure how to do it. Let's think through this sometime together, and then circle back & add it